app/Nova/Resources/Integration.php

```diff
@@ -178,7 +178,7 @@ public function actions(NovaRequest $request): array
                     return $model->status === IntegrationStatus::Draft->value;
                 }),
 
-            (new OpenWidgetBeheer())
+            (new OpenWidgetManager())
```

Comment:
Is the label of the custom Nova action still "Open Widget Beheer" or did it become "Open Widget Manager"?

Reply:
I left if as Beheer yes

Reply:
I was confused, the `$name` is defined inside the action itself and not here.